### DISCUSSION TITLE
CONTRIBUTING.md missing folder crna-kitchen-sink

### DISCUSSION THREAD

**Discussion by CezarCobuz** [storybookjs/react-native#335] [Q&A] [answered]
I was going through the initial setup from [CONTRIBUTING.md](https://github.com/storybookjs/react-native/blob/master/CONTRIBUTING.md#expo-app) but at the `Expo App` step there is no `crna-kitchen-sink` folder within `examples`. I only see the `native` folder. What did I missed?

**Comment by dannyhw** [comment, marked as answer]
@CezarCobuz the expo example was removed to simplify the setup and reduce the size of the repo. You can use the examples/native app instead. I think if you look in the next-6.0 branch this part of the contributing docs was already changed.

**Reply by CezarCobuz** [reply]
Yes, you are right, the link at the bottom of README.md (..Read our [contributing guide](https://github.com/storybookjs/react-native/blob/master/CONTRIBUTING.md) to get started..) points to [master contributing guide](https://github.com/storybookjs/react-native/blob/master/CONTRIBUTING.md). Is that intended?

**Reply by dannyhw** [reply]
No that should be updated, thanks for pointing that out!
